examples/platforms/simulation/platform-simulation.h

```diff
@@ -66,6 +66,8 @@ enum
     OT_SIM_EVENT_RADIO_RECEIVED     = 1,
     OT_SIM_EVENT_UART_WRITE         = 2,
     OT_SIM_EVENT_RADIO_SPINEL_WRITE = 3,
+    OT_SIM_EVENT_POST_CMD           = 4,
+    OT_SIM_EVENT_STATUS_PUSH        = 5,
```

Comment:
```suggestion
    OT_SIM_EVENT_OTNS_STATUS_PUSH   = 5,
```

For a new reader, I'd like to know more context about this event (`STATUS` would be too general). Besides, this event is introduced by OTNS and used by `OTNS` exclusively, I think it is better to add this `OTNS` prefix. Thoughts?

We also need a clearer name for `OT_SIM_EVENT_POST_CMD`, but since it is not introduced by OTNS, we can leave it for future enhancements.

Reply:
agree.